### PR TITLE
csgo-to-cs2 - Csgo renamed to CS2

### PR DESCRIPTION
**Pull request by kafkapre**

### CODE REVIEW

File: proto/bragi/bragi_service.proto [hunk 1]
```diff
@@ -95,8 +95,8 @@ message MatchMessage {
 // it is not necessary to use it for match state maintain/reconstruction.
 message MatchSnapshot {
   oneof snapshot {
-    CsgoMatchSnapshot csgo = 1;
-    CsgoMatchSnapshot csgo_wingman = 2;
+    CS2MatchSnapshot cs2 = 1;
+    CS2MatchSnapshot cs2_1v1 = 2;
```
Comment (pmachan) [inline proto/bragi/bragi_service.proto:99]: cs2_1v1 -> cs2_duel

File: proto/bragi/bragi_service.proto [hunk 2]
```diff
@@ -109,8 +109,8 @@ message MatchSnapshot {
 // it is not necessary to use it for match state maintain/reconstruction.
 message MatchState {
   oneof state {
-    CsgoMatchState csgo = 1;
-    CsgoMatchState csgo_wingman = 2;
+    CS2MatchState cs2 = 1;
+    CS2MatchState cs2_1v1 = 2;
```
Comment (pmachan) [inline proto/bragi/bragi_service.proto:113]: cs2_1v1 -> cs2_duel

**Comment by pmachan** [comment]
Its OK now

**Review by pmachan — APPROVED** [review]

**Review by Radek-Pysny — APPROVED** [review]
It is OK if we are OK with breaking API by simply renaming everything.

**Pull request merged**